packages/pdf-viewer/hooks/useScrollSaver.ts

```diff
@@ -1,15 +1,20 @@
-import { useRef, useEffect, MutableRefObject } from 'react';
-import { ScaledSize } from '../pdfSource';
+import { useRef, useEffect, MutableRefObject, useState } from 'react';
+import { ScaledSize, PdfData } from '../pdfSource';
 
 export interface ScrollSaver {
 	container: MutableRefObject<HTMLElement>;
 	scaledSize: ScaledSize;
 	pdfId: string;
 	rememberScroll: boolean;
+	onActivePageChange?: (activePage: number)=> void;
```

Comment:
Make "onActivePageChange" required. As much as possible avoid optional stuff, especially when it's not really optional in your code (since you always set it).